src/System.Memory/tests/TestHelpers.cs

```diff
@@ -294,6 +294,53 @@ public enum TestEnum
             e4,
         }
 
+        public struct Length24
+        {
+            public ulong a1;
+            public ulong a2;
+            public ulong a3;
+        }
+
+        public struct Length24_2
+        {
+            public long a1;
+            public long a2;
+            public long a3;
+        }
+
+        public struct ThreeByteStruct
+        {
+            public byte a1;
+            public byte a2;
+            public byte a3;
+        }
+
+        public struct PaddedStruct4
+        {
+            public byte a1;  // 1 bytes
+                             // 1 bytes padding
+            public char a2;  // 2 bytes
+
+            public PaddedStruct4(byte a11, char a22)
+            {
+                a1 = a11;
+                a2 = a22;
+            }
+        }
+
+        public struct PaddedStruct
+        {
+            public char a1; // 2 bytes
+                            // 2 bytes padding
+            public int a2;  // 4 bytes
```

Comment:
Is this struct guaranteed to be laid out like this or do we need to set explicit offsets, for example:
https://github.com/dotnet/corefx/blob/master/src/System.Memory/tests/TestHelpers.cs#L241